### PR TITLE
Symantec mss invalid characters fix

### PR DESCRIPTION
**Pull request by orlichter1**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/21237

## Description
When fetching from Symantec, some "blank" characters were imported as well (end of word chars and such). We removed them.

## Screenshots
Paste here any images that will help the reviewer

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by orlichter1** [comment]
@yaakovi How can I revert changes only to conf.json?

File: Integrations/SymantecMSS/SymantecMSS.py [hunk 1]
```diff
@@ -74,15 +77,20 @@ def load_severities():
 ''' HELPER FUNCTIONS '''
 
 
+def strip_unwanted_chars(s):
+    return re.sub('&.+[0-9]+.?;', '', s)
```
Comment (yaakovi) [inline Integrations/SymantecMSS/SymantecMSS.py]: as you don't have any match group here this will do nothing

**Comment by yaakovi** [comment]
still need help with conf.json?
you can use `git revert` with file parameter

File: Integrations/SymantecMSS/SymantecMSS.py (hunk 1 above)
Comment (orlichter1) [reply Integrations/SymantecMSS/SymantecMSS.py]: @yaakovi What do you mean? This takes all &4954; words and removes them from s. This is exactly what fixes the bug

File: Integrations/SymantecMSS/SymantecMSS_test.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# coding=utf-8
+import demistomock as demisto
+
+
+def test_bug(mocker, requests_mock):
```
Comment (yaakovi) [inline Integrations/SymantecMSS/SymantecMSS_test.py]: switch to a better name

File: TestData/SymantecXML.txt [hunk 3]
```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="utf-8"?><soap:Envelope xmlns:soap="www.example.com" xmlns:xsi="www.example.com" xmlns:xsd="www.example.com"><soap:Body><IncidentWorkflowQueryResponse xmlns="www.example.com"><IncidentWorkflowQueryResult><SecurityIncident xmlns:xsd="www.example.com" xmlns:xsi="www.example.com" xmlns=""><IncidentNumber>175726938</IncidentNumber><TimeCreated>2020-01-09T18:44:56.417</TimeCreated><Correlation>No</Correlation><Severity>Warning</Severity><Classification>Server Attack</Classification><Description>This incident represents an attempted exploit against a server on your network.
```
Comment (yaakovi) [inline TestData/SymantecXML.txt]: move to integration folder, `Integrations/SymantecMSS/test_data`

File: Integrations/SymantecMSS/CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 ## [Unreleased]
-
+Resolved issue caused by data with empty characters from Symantec MSS.
```
Comment (yaakovi) [inline Integrations/SymantecMSS/CHANGELOG.md]: Fixed an issue where ***fetch-incidents*** failed on data containing special characters.

**Comment by kirbles19** [comment]
@orlichter1 Perfect, no change.

**Pull request merged**